subsys/net/ip/tcp2.c

```diff
@@ -1324,7 +1327,7 @@ static void tp_init(struct tcp *conn, struct tp *tp)
 	struct tp out = {
 		.msg = "",
 		.status = "",
-		.state = tcp_state_to_str(conn->state, true),
+		.state = tcp_state_to_str(conn->state),
```

Comment:
There's a problem with this on TTCN-3 side, it needs the state with the prefix.

So, if you don't mind, let's leave tcp_state_to_str() as is.

Reply:
We need to figure out something, the logging functions do not work properly with static variables. I was seeing weird debug prints.

Reply:
Lets drop that patch temporarily in order to continue with ttcn-3. I will send a new version.